deb/files/changelog

```diff
@@ -1,3 +1,10 @@
+firefish (20240330-d202404010001+release) unstable; urgency=high
```

Comment:
Please use `urgency=medium`
https://www.debian.org/doc/manuals/developers-reference/best-pkging-practices.html#selecting-the-upload-urgency